extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -861,7 +861,7 @@ public enum ResponseMode {
          * However, if multiple authentications are attempted from the same browser, for example, from the different
          * browser tabs, then the currently available state cookie may represent the authentication flow
          * initiated from another tab and not related to the current request.
-         * Disable this property if you would like to avoid supporting multiple authorization code flows running in the same
+         * Disable this property if you want to avoid supporting multiple authorization code flows running in the same
```

Comment:
It feels weird.. "disable to avoid". I think this is more of a "disable to constrain" situation. The effect of disabling is that you can only have a single authorization code flow, IIUC.

```suggestion
         * Disable this property to permit only a single authorization code flow in the same
```